### PR TITLE
backend-tests: use existing device for creating single device deployment

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
after this https://github.com/mendersoftware/deployments-enterprise/pull/507 fix, the deployments will return error if user will try to create single device deployment for non-existent device

### CODE REVIEW

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

File: backend-tests/tests/test_auditlogs.py [hunk 1]
```diff
@@ -131,7 +131,12 @@ def test_deployment_create(self, tenant_users):
         """Baseline test - deployment create event is logged with correct fields."""
         user = tenant_users.users[0]
 
-        d, _ = make_deployment(user.token)
+        devauthd = ApiClient(deviceauth.URL_DEVICES)
```
Comment (MaciejTe) [inline backend-tests/tests/test_auditlogs.py:134]: You can use `self.devauthd` from `setup` method

File: backend-tests/tests/test_auditlogs.py [hunk 2]
```diff
@@ -226,6 +231,11 @@ def get_auditlog_time(oid: str):
             else:
                 assert False, "max GET /logs retries hit"
 
+        devauthd = ApiClient(deviceauth.URL_DEVICES)
```
Comment (MaciejTe) [inline backend-tests/tests/test_auditlogs.py:234]: You can use self.devauthd and self.devauthm from setup method

File: backend-tests/tests/test_auditlogs.py [hunk 3]
```diff
@@ -241,7 +251,7 @@ def get_auditlog_time(oid: str):
                 evt = evt_user_create(user, uid, uuidv4 + "@acme.com")
                 oid = uid
             else:
-                d, a = make_deployment(user.token)
+                d, a = make_deployment(user.token, device.id)
```
Comment (MaciejTe) [inline backend-tests/tests/test_auditlogs.py:254]: Nitpick: I'd use opportunity and consider renaming `d` and `a` variables to something more understandable ;-)

File: backend-tests/tests/test_auditlogs.py [hunk 4]
```diff
@@ -425,7 +435,7 @@ def _test_args_sort(self, tenant_users, events):
                 check_log(resp[i], case["expected"][i])
 
 
-def make_deployment(token: str) -> Tuple[Dict, Dict]:
+def make_deployment(token: str, device: str = "arm1") -> Tuple[Dict, Dict]:
```
Comment (MaciejTe) [inline backend-tests/tests/test_auditlogs.py:438]: Please consider renaming this argument to `device_id` - why its default value is `arm1`? It looks more like device type, not device ID.

**Review by alfrunes — APPROVED** [review]
Except for the nitpicks @MaciejTe pointed already out, this LGTM (pre-approving).

**Comment by MaciejTe** [comment]
Pre-approving too

**Review by MaciejTe — APPROVED** [review]

**Comment by tranchitella** [comment]
I think this is superseded by https://github.com/mendersoftware/integration/pull/1788

**Pull request closed without merging**